notes-webapp/src/main/webapp/vue-app/notes-extensions/components/favorites/NotesFavoriteItem.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <v-list-item class="clickable" :href="noteUrl">
     <v-list-item-icon class="me-3 my-auto">
-      <v-icon size="22" class="icon-default-color"> fas fa-clipboard </v-icon>
+      <v-img
+        :src="noteImg"
+        max-height= "28"
+        max-width= "25" />
```

Comment:
Can you reduce the image size please in Bytes knowing that its height and width will be very small comparing to its resolution. @Samuel-Renault-eXo ( To check with Sofien please)

Reply:
Last proposed and validated image here
![notes-appicon](https://user-images.githubusercontent.com/58853195/175274513-7cb8591f-3210-466a-8d01-89cb45e55a0f.png)

Reply:
ok, thanks